### PR TITLE
CPlayerInfo done

### PR DESCRIPTION
**Pull request by erorcun**

### CODE REVIEW

File: src/core/PlayerInfo.cpp [hunk 1]
```diff
@@ -113,8 +118,8 @@ CPlayerInfo::Clear(void)
 	m_nTrafficMultiplier = 0;
 	m_fRoadDensity = 1.0f;
 	m_bInRemoteMode = false;
-	m_bSwitchTaxi = false;
-	m_nSwitchTaxiTime = 0;
+	m_bUnusedTaxiThing = false;
```
Comment (Nick007J) [inline src/core/PlayerInfo.cpp]: There is a script command SWITCH_TAXI_TIMER that uses those fields, don't think it was good either way.

File: src/core/PlayerInfo.cpp [hunk 2]
```diff
@@ -306,6 +311,256 @@ INITSAVEBUF
...
+				}
+			}
+		}
+	}
+}
+
+void
+CPlayerInfo::Process(void)
+{
+	// Unused taxi feature. Gives you a dollar for every second with a passenger. Can be toggled via 0x29A opcode.
```
Comment (Nick007J) [inline src/core/PlayerInfo.cpp]: Never mind you mention it :D

File: src/core/PlayerInfo.cpp [hunk 3]
```diff
@@ -306,6 +311,256 @@ INITSAVEBUF
...
+	if (startTaxiTimer)
+		m_nUnusedTaxiTimer = CTimer::GetTimeInMilliseconds();
+
+	// The effect that makes money counter does while earning/losing money
+	if (m_nVisibleMoney != m_nMoney) {
+		int diff = m_nMoney - m_nVisibleMoney;
+		int diffAbs = Abs(diff);
+		int changeBy;
+
+		if (diffAbs <= 100000) {
```
Comment (Nick007J) [inline src/core/PlayerInfo.cpp]: Almost certain it was
```
if (diffAbs > 100000)
  changeBy = 12345;
else if (diffAbs > 10000)
  changeBy = 1234;
...
```
It looks like that even in assembly, not sure why IDA outputs it like that

Comment (erorcun) [reply src/core/PlayerInfo.cpp]: ok

File: src/core/PlayerInfo.cpp (hunk 2 above)
Comment (erorcun) [reply src/core/PlayerInfo.cpp]: :)

**Pull request merged**